service/operator/service.go

```diff
@@ -27,31 +26,12 @@ type Config struct {
 // DefaultConfig provides a default configuration to create a new service by
 // best effort.
 func DefaultConfig() Config {
-	var err error
-
-	var k8sClient kubernetes.Interface
-	{
-		config := k8s.DefaultConfig()
-		k8sClient, err = k8s.NewClient(config)
-		if err != nil {
-			panic(err)
-		}
-	}
-
-	var newLogger micrologger.Logger
-	{
-		config := micrologger.DefaultConfig()
-		newLogger, err = micrologger.New(config)
-		if err != nil {
-			panic(err)
-		}
-	}
-
```

Comment:
Boyscouting here. Should have done this separately. Thanks for bearing with me. <3